src/schemas.py

```diff
@@ -1,10 +1,33 @@
 import json
 from datetime import datetime
+from enum import Enum
```

Comment:
I think we can remove all changes from this file. I just confirmed on `master` we have all of this stuff in `models.py`. As long as everything still runs without the changes in this file, we can safely remove